astro/src/content/docs/apis/_import-users-request-body.mdx

```diff
@@ -141,12 +141,16 @@ You must provide either the **email** or the **username** field for each User. T
   </Aside>
 </APIField>
 
-<APIField name="users[x].password" type="String" required>
+<APIField name="users[x].password" type="String" optional>
   The User's password. If <InlineField>encryptionScheme</InlineField> is provided this value is the existing password hash, if <InlineField>encryptionScheme</InlineField> is not provided this value is assumed to be plain text and it will be hashed during import.
 
   The value of <InlineField>password</InlineField> should contain only the password hash, or, if the <InlineField>encryptionScheme</InlineField> is omitted, the plain text password.  Unless you are using a custom plugin that does not utilize base64, <InlineField>password</InlineField> should be base64 encoded.  It is recommended to use a base64 encoded string, but if you are importing hashes from a legacy system that uses base16, base32, or another encoding and are writing a custom plugin, import the hash in the same format the plugin produces.
 
   Please see the [Password Hashes](#password-hashes) section for more information about importing particular password hashes.
+
+  <Aside>
```

Comment:
[Dan is re-working](https://github.com/FusionAuth/fusionauth-site/pull/3665/files#diff-c2d193edc4873b181b08d47db4077538ee92cb02f6d58d11c90caf52f31386d3R234) some of these call outs. Use
```suggestion
  <Aside title="Available since 1.99.9" type="version">
```
or similar to keep up with that change.

Reply:
@spwitt incorporated, although trimmed 'Available' since that doesn't seem to make sense here.